README.md

```diff
@@ -443,6 +443,7 @@ Name | Website | Region
 [npm](/company-profiles/npm.md) | https://www.npmjs.com/ | USA
 [Nuage](/company-profiles/nuage.md) | https://nuagebiz.tech/ | India
 [Nuna](/company-profiles/nuna.md) | https://www.nuna.com/ | USA
+[Ocient](/company-profiles/ocient.md) | https://ocient.com | USA, Europe, India
```

Comment:
`Company is listed out of order: "O'Reilly Media" (should be before "Ocient")` The logs are now saying that Oncient should be after `Oncient` although earlier it said they had to be after 😆